homeassistant/components/thread/config_flow.py

```diff
@@ -15,9 +15,7 @@ class ThreadConfigFlow(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    async def async_step_import(
-        self, import_data: dict[str, str] | None = None
-    ) -> ConfigFlowResult:
+    async def async_step_import(self, import_data: None) -> ConfigFlowResult:
```

Comment:
No typing?

Reply:
> No typing?

There is typing - but for thread it is always `None`